src/welcome/assets/components/TourPage.tsx

```diff
@@ -65,72 +65,80 @@ export default class TourPage extends Component<{
                     </Col>
                 </Row>
             </Container>
-        )
+        );
     }
 
     showStep = (newStep: number) => {
         this.setState({
             step: newStep
-        })
+        });
     }
 
     nextStep = () => {
         const newStep = this.state.step + 1;
         this.setState({
             step: newStep
-        })
+        });
     }
 
     getContentPages = () => {
         /**
-         * TODO: 
          * Provide 4 pages:
          * 1. open folder: for full features
          * 2. project view
          * 3. run/debug: point out the entry we want to promote
          * 4. testing
-         * 
-         * Polish screenshots to highlight entries.
          */
 
+        const openFolderCommand: string = encodeCommandUriWithTelemetry("open folder", supportedByNavigator("mac") ? "workbench.action.files.openFileFolder" : "workbench.action.files.openFolder");
+        const focusOnProjectExplorerCommand: string = encodeCommandUriWithTelemetry("focus on Project Explorer", "javaProjectExplorer.focus");
```

Comment:
Why only this command is marked as `focus` and others are `show`?

Reply:
I'll change all to `showXXX`